src/js/grapl-cdk/swarm/swarm_setup.py

```diff
@@ -222,8 +222,8 @@ def main(prefix: str) -> None:
 export AWS02_NAME="{worker_hostnames[0]}"
 export AWS03_NAME="{worker_hostnames[1]}"
 
-aws s3 cp s3://$GRAPL_DEPLOY_NAME-dgraph-config-bucket/docker-compose-dgraph.yml .
-aws s3 cp s3://$GRAPL_DEPLOY_NAME-dgraph-config-bucket/envoy.yaml .
+aws s3 cp s3://${{GRAPL_DEPLOY_NAME,,}}-dgraph-config-bucket/docker-compose-dgraph.yml .
```

Comment:
what are the commas for here?

Reply:
Those will have bash (the assumed shell) change all alpha chars to lowercase. This is needed to refer to these S3 buckets when the deployment name has capital letters in it.